plotter/main.go

```diff
@@ -49,22 +51,61 @@ var examples = []struct {
 }
 
 var formats = []string{
-	".eps",
-	".pdf",
-	".svg",
-	".png",
-	".tiff",
-	".jpg",
+	"eps",
+	"pdf",
+	"svg",
+	"png",
+	"tiff",
+	"jpg",
+}
+
+type canvasWriter interface {
+	vg.CanvasSizer
+	io.WriterTo
 }
 
 func main() {
-	for _, ex := range examples {
-		for _, f := range formats {
-			err := ex.p.Save(4*vg.Inch, 4*vg.Inch, ex.name+f)
-			if err != nil {
-				log.Fatalf("failed to save %s%s: %v", ex.name, f, err)
+	//for _, ex := range examples {
+	//	for _, f := range formats {
+	//		err := ex.p.Save(4*vg.Inch, 4*vg.Inch, ex.name+"."+f)
+	//		if err != nil {
+	//			log.Fatalf("failed to save %s.%s: %v", ex.name, f, err)
+	//		}
+	//	}
+	//}
+	for _, f := range formats {
+		c, err := draw.NewFormattedCanvas(4*5*vg.Inch, 4*4*vg.Inch, f)
```

Comment:
perhaps lift and use the `nrows` and `ncols` constants here instead of these magic `4` and `5` litterals?